File: core/core-api/src/main/java/com/coffee/gu/api/controller/v1/request/ModifyCartItemRequest.java
package com.coffee.gu.api.controller.v1.request;

import com.coffee.gu.cart.ModifyCartItem;

public record ModifyCartItemRequest(
        Long quantity
) {
    public ModifyCartItem toModifyCartItem(Long cartItemId) {
        return new ModifyCartItem(cartItemId, quantity);
    }
}

File: core/core-api/src/main/kotlin/com/coffee/gu/api/controller/v1/request/ModifyCartItemRequest.kt
package com.coffee.gu.api.controller.v1.request

import com.coffee.gu.cart.ModifyCartItem

class ModifyCartItemRequest(
    val quantity: Long,
) {
    fun toModifyCartItem(cartItemId: Long): ModifyCartItem {
        return ModifyCartItem(cartItemId, quantity)
    }
}
